examples/medplum-live-chat-demo/vite.config.ts

```diff
@@ -11,4 +12,11 @@ export default defineConfig({
     host: 'localhost',
     port: 3000,
   },
+  resolve: {
+    alias: {
+      '@medplum/core': path.resolve(__dirname, '../../packages/core/src'),
```

Comment:
This is useful during development, but we can't commit it / can't publish it to the example repo.  Not sure if there's a way to make this conditional?  Simple option is to just remove before finalizing.

Reply:
I'll remove them for now and we can add them back in a separate PR and maybe add a step to remove them when syncing the example repos? Just saves so much time